transform/mattermost-analytics/models/intermediate/product/nps/int_nps_server_version_spined.sql

```diff
@@ -0,0 +1,27 @@
+WITH first_server_version AS (
+    SELECT 
+        server_id,
+        server_version,
+        MIN(event_date) AS min_event_date
+    FROM {{ ref('int_nps_score') }}
+    WHERE event_date >= '{{ var('nps_start_date')}}'
+    GROUP BY all
+),
+spined AS (
+    SELECT 
+        date_day::date AS activity_date,
+        server_id AS server_id,
+        case when date_day::date = min_event_date then server_version end as server_version
+    FROM first_server_version fsd
+    LEFT JOIN {{ ref('nps_days') }} td 
+        ON date_day::date >= min_event_date
```

Comment:
The previous CTE will result in multiple date/server id combinations in this CTE as well.

Reply:
Yes, this is now updated so this multiple combinations don't occur.